livechat/overview/get_started.rst

```diff
@@ -0,0 +1,83 @@
+==========================
+Get Started with Live Chat
+==========================
+
+Live chat is a cost-effective tool and a great way to bring visitors back to your website. It allows
+fast responses, human interaction, sales' improvement, accessible and convenient support, and even
+the reduce of services costs. Customers want to talk to you, so let's make it easy.
+
+Activate and set it up
+======================
+
+Once *Live Chat* is installed on your database, if your website was created with Odoo, the
+application is automatically added to it. All that is left to do is to go to
+:menuselection:`Website --> Configuration --> Settings --> Live Chat`.
+
+.. image:: media/livechat_settings.png
+   :align: center
+   :height: 300
+   :alt: View of the settings page and the live chat feature for Odoo Live Chat
+
+Select the channel to be linked to your website or create one on the fly.
+
+.. image:: media/channels.png
+   :align: center
+   :alt: View of a live chat channel form for Odoo Live Chat
+
+| For both scenarios, under:
+| - **Operators**: add agents to respond to the chat requests. Add as many as you like, and keep in
+  mind that operators that do not show any activity in Odoo for more than 30min are considered
+  disconnected.
+| - **Options**: set the default text to be shown on the live chat button; an automated welcome
+  message to be seen by visitors when a conversation is initiated, and the text that prompts the
+  user to initiate a chat.
+| - **Channel Rules**: choose an action for a given URL, and/or per country. In the example below,
+  the chat window pops-up automatically after 3 seconds of users, from any country, landing on the
```

Comment:
Not sure to understand "3 seconds of users".

Reply:
I think it's the ", from any country," that lost you.
Keli says that "The window pops-up after the user lands 3 seconds on the page".

Reply:
Take a look. I believe it is clearer.